content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -83,10 +83,46 @@ a different label key if you prefer.
 
 ## Automatic node labelling {#node-labeller}
 
-If you're using AMD GPU devices, you can deploy
-[Node Labeller](https://github.com/RadeonOpenCompute/k8s-device-plugin/tree/master/cmd/k8s-node-labeller).
-Node Labeller is a {{< glossary_tooltip text="controller" term_id="controller" >}} that automatically
-labels your nodes with GPU device properties.
+As an administrator, you can automatically discover and label all your GPU enabled nodes
+by deploying Kubernetes [Node Feature Discovery](https://github.com/kubernetes-sigs/node-feature-discovery) (NFD).
+NFD detects the hardware features that are available on each node in a Kubernetes cluster.
+Typically, NFD is configured to advertise those features as node labels, but NFD can also add extended resources, annotations, and node taints.
+NFD is compatible with all [supported versions](/releases/version-skew-policy/#supported-versions) of Kubernetes.
 
-Similar functionality for NVIDIA is provided by
-[GPU feature discovery](https://github.com/NVIDIA/gpu-feature-discovery/blob/main/README.md).
+Administrators can leverage NFD to also taint nodes with specific features, so that only pods that request those features can be scheduled on those nodes.
+After a cluster is labeled with the GPU feature, you can schedule pods on GPU nodes by adding the following to your pod spec:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example-vector-add
+spec:
+  # You can then use Kubernetes node affinity to schedule pods on GPU nodes.
+  # https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: "gpu.gpu-vendor.example/installed-memory"
+            operator: Gt
+            values: ["40535"]
+          - key: "gpu.gpu-vendor.example/family"
+            operator: In
+            values:
+            - Helium # example product family
+            - Neon   # example product family
+  restartPolicy: Never
+  containers:
+    - name: example-vector-add
+      image: "registry.example/example-vector-add:v42"
+      resources:
+        limits:
+          gpu-vendor.example/example-gpu: 1 # requesting 1 GPU
+```
+
+#### GPU vendor implementations
+
+- [NVIDIA](https://github.com/NVIDIA/gpu-feature-discovery/#readme).
+- [Intel](https://github.com/intel/intel-device-plugins-for-kubernetes).
```

Comment:
Alphabetical order please
```suggestion

These vendors provide labelling implementations that integrate with NFD for GPU hardware discovery:

- [Intel](https://github.com/intel/intel-device-plugins-for-kubernetes).
- [NVIDIA](https://github.com/NVIDIA/gpu-feature-discovery/#readme).
```

Reply:
As this is only about GPUs, it's better to refer directly to Intel GPU plugin documentation: https://intel.github.io/intel-device-plugins-for-kubernetes/cmd/gpu_plugin/README.html